app/models/form_profiles/va_686c674.rb

```diff
@@ -34,7 +34,13 @@ def metadata
   private
 
   def prefill_form_address
-    mailing_address = VAProfileRedis::ContactInformation.for_user(user).mailing_address if user.vet360_id.present?
+    redis_prefill = if Flipper.enabled?(:remove_pciu,
+                                        user)
+                      VAProfileRedis::V2::ContactInformation.for_user(user)
+                    else
+                      VAProfileRedis::ContactInformation.for_user(user)
+                    end
+    mailing_address = redis_prefill.mailing_address if user.vet360_id.present?
```

Comment:
We should probably use the safe method chaining here. It'll return nil then instead of raising a no method error

```
redis_prefill&.mailing_address
```